src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,142 @@
 package core.basesyntax;
 
+import static java.lang.Math.abs;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private final int DEFAULT_INITIAL_CAPACITY = 16;
+    private final double DEFAULT_LOAD_FACTOR = 0.75;
+    private HashObject<K, V>[] hashMap;
+    private int capacity = DEFAULT_INITIAL_CAPACITY;
+    private int size = 0;
+
+    public MyHashMap() {
+        hashMap = new HashObject[DEFAULT_INITIAL_CAPACITY];
+    }
+
+    public class HashObject<K, V> {
```

Comment:
According to the checklist, private methods and classes should be placed after public ones in your class. Therefore, the HashObject class should be moved down, after all public methods.